src/stories/modal-loan/ModalLoan.tsx

```diff
@@ -0,0 +1,236 @@
+  title: string;
+  description: string;
+  showModal: boolean;
+  showExpired: boolean;
+};
+
+export const ModalLoan = (props: ModalLoanProps) => {
+  const [showModal, setShow] = useState(props.showModal);
+
+  useEffect(() => {
+    setShow(props.showModal);
+  }, [props.showModal]);
+
+  const toggleModal = () => {
+    setShow(!showModal);
+  };
+
+  const isExpired = props.showExpired;
+  const loanList = isExpired ? listExpired : listExpiresSoon;
+
+  const countRenewable = useMemo(() => {
+    let total = 0;
+    loanList.forEach((loans) =>
+      loans.list.forEach((material) => {
+        if (material.canBeRenewed) total += 1;
+      })
+    );
+    return total;
+  }, [loanList]);
+
+  if (!showModal) {
+    return <ModalFallbackButton toggleModal={toggleModal} />;
+  }
+
+  return (
+    <div
+      className={`modal modal-padding modal-loan ${
+        showModal ? "modal-show" : ""
+      }`}
+    >
+      <ModalCloseButton toggleModal={toggleModal} />
+      <div className="modal-loan__container">
+        <div className="modal-loan__header">
+          {isExpired && (
+            <div className="mr-32">
+              <Counter
+                label={"dage"}
+                percentage={100}
+                value={-2}
+                status={"danger"}
+                isReady={false}
+              />
+            </div>
+          )}
+          <div>
+            <h2 className="modal-loan__title text-header-h2">{props.title}</h2>
+            <p className="text-body-medium-regular color-secondary-gray mt-4">
+              {props.description}
+            </p>
+          </div>
+        </div>
+        {isExpired && (
+          <div className="modal-loan__warning mt-48 mb-48">
+            <WarningStatus
+              url="/"
+              description="Afleveringsdatoen for lånet er overskredet, derfor pålægges du et gebyr, når materialet afleveres"
+            />
+          </div>
+        )}
+
+        <div className="modal-loan__buttons">
+          <Button
+            buttonType="default"
+            label={`Forny mulige (${countRenewable})`}
+            size="small"
+            variant="filled"
+            disabled={false}
+            collapsible={true}
+          />
+        </div>
+
+        <div className="modal-loan__list__container">
+          {loanList.map(({ materialType, list }) => (
+            <div className="modal-loan__list">
+              {materialType && (
+                <div className="modal-loan__count">
+                  <LinkFilters
+                    filters={[
+                      {
+                        title: materialType,
+                        counter: `${list.length}`,
+                      },
+                    ]}
+                  />
+                </div>
+              )}
+              <div className="modal-loan__list__materials">
+                {list.map((listItem, index) => (
+                  <ListMaterials key={index} {...listItem} />
```

Comment:
To maintain a good markup structure I suggest making this an (un)ordered list with each material being wrapped by a list item.

Reply:
👍 done